internal/host/host.go

```diff
@@ -497,7 +497,7 @@ func (m *Manager) reportInstallationMetrics(ctx context.Context, h *models.Host,
 	if err != nil {
 		log.WithError(err).Errorf("not reporting installation metrics - failed to find cluster %s", h.ClusterID)
 	} else {
-		m.metricApi.ReportHostInstallationMetrics(log, cluster.OpenshiftVersion, h, previousProgress, CurrentStage)
+		m.metricApi.ReportHostInstallationMetrics(log, cluster.OpenshiftVersion, *cluster.ID, h, previousProgress, CurrentStage)
```

Comment:
@gamli75 , @romfreiman , @avishayt  if we have the clusterid for the host metrics, don't we also need hostId?

Reply:
@ronniel1 I talked with @romfreiman about it and we agreed that we can add it later. The host HW is more important (VM ..)